packages/graphql-transformers-e2e-tests/src/__tests__/IndexWithAuthV2WithFF.e2e.test.ts

```diff
@@ -314,22 +309,23 @@ async function createFamilyMember(client: GraphQLClient, parent: string, child:
 async function listFamilyMembers(client: GraphQLClient, args?: Record<string, any>) {
   const result = await client.query(
     `query ListFamilyMembers(
+      $id: ModelIDKeyConditionInput
       $parent: ID
-      $child: ModelIDKeyConditionInput
```

Comment:
Is there a reason to remove this field from an existing test?

Reply:
Yes, `$child` is no longer part of the primary key's query.